flashinfer_bench/compile/builder.py

```
@@ -127,7 +127,11 @@ def _get_package_name_and_build_path(self, solution: Solution) -> Tuple[str, Pat
         return package_name, build_path
 
     def _try_validate_signature(
-        self, callable: Callable, definition: Definition, solution: Solution
+        self,
+        callable: Callable,
```

Ruff (0.15.21)

[error] Function argument `callable` is shadowing a Python builtin

(A002)